bundler/doc/contributing/ISSUES.md

```diff
@@ -46,6 +46,6 @@ If your version of Bundler does not have the `bundle env` command, then please i
 
 If you have either `rubygems-bundler` or `open_gem` installed, please try removing them and then following the troubleshooting steps above before opening a new ticket.
 
-[Create a gist](https://gist.github.com) containing all of that information, then visit the [Bundler issue tracker](https://github.com/rubygems/bundler/issues) and [create a ticket](https://github.com/rubygems/bundler/issues/new) describing your problem and linking to your gist.
+[Create a gist](https://gist.github.com) containing all of that information, then visit the [Bundler issue tracker](https://github.com/rubygems/rubygems/issues) and [create a ticket](https://github.com/rubygems/rubygems/issues/new?labels=Bundler&template=bundler-related-issue.md) describing your problem and linking to your gist.
```

Comment:
I’ve added two parameters so that the issue is automatically labeled “Bundler” and the [Bundler-related template](https://github.com/rubygems/rubygems/blob/master/.github/ISSUE_TEMPLATE/bundler-related-issue.md) is presented.